NCS.DSS.ContentEnhancer.Tests/NCS.DSS.ContentEnhancer.Tests.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
```

Comment:
Confirm if all attributes in this group are necessary, as they differ to other .NET 8 repos
@amjones99 @PradeepVoorukonda